app/frontend/schools/settings/components/SchoolCustomize__LinkComponent.res

```diff
@@ -127,18 +127,29 @@ module LinkEditor = {
       {switch kind {
       | HeaderLink
       | FooterLink => <>
-          <input
-            value=state.title
-            required=true
-            autoFocus=true
-            id={"link-title-" ++ id}
-            className=inputClasses
-            placeholder="A short title for a new link"
-            onChange={event => {
-              let value = ReactEvent.Form.target(event)["value"]
-              send(UpdateTitle(value))
-            }}
-          />
+          <div className="flex flex-col gap-1 flex-1">
+            <input
+              value=state.title
+              required=true
+              autoFocus=true
+              id={"link-title-" ++ id}
+              className=inputClasses
+              placeholder="A short title for a new link"
+              onChange={event => {
+                let value = ReactEvent.Form.target(event)["value"]
+                send(UpdateTitle(value))
+              }}
+              maxLength=24
+            />
+            <School__InputGroupError
+              active={!StringUtils.isPresent(state.title)} message={t("cant_empty_message")}
+            />
+            <School__InputGroupError
+              active={state.title->String.length == 24}
```

Comment:
What happens if the `state.title` is more than `24` characters? (I think you can trigger this by tweaking the `maxLength` in HTML`

Reply:
> What happens if the state.title is more than 24 characters?

The `max_length` attribute is set to `24`. So, when the character length reaches `24` it displays the warning message `Reached maximum length of 24 characters`.

As the `max_length` is set to `24` user cannot enter more than 24 characters.